yaml/github-actions/security/workflow-run-target-code-checkout.yaml

```diff
@@ -14,7 +14,7 @@ rules:
     scripts (e.g., `python setup.py install`).
     Audit your workflow file to make sure no code from the incoming PR is executed.
     Please see https://securitylab.github.com/research/github-actions-preventing-pwn-requests/ for additional
-    mitigations.
+    mitigations. More information available at https://www.legitsecurity.com/blog/github-privilege-escalation-vulnerability
```

Comment:
Can you please move it to `references:` metadata field? (here https://github.com/returntocorp/semgrep-rules/blob/74647f4b9fe3e13c882b902b7b6680deadc48e50/yaml/github-actions/security/workflow-run-target-code-checkout.yaml#L23) I think it would make more sense since in the message we mention official resource with mitigation description, while `https://www.legitsecurity.com/blog/github-privilege-escalation-vulnerability` is a bug hunting write up